package.json

```diff
@@ -9,7 +9,7 @@
     "bootstrap": "lerna bootstrap",
     "prepare": "if test \"$NODE_ENV\" != \"production\" && test \"$CI\" != \"true\" ; then husky install ; fi",
     "prepack": "lerna run --stream prepack",
-    "dev": "lerna run --stream prepack && lerna --scope=@logto/{core,ui,console,demo-app} exec -- pnpm dev",
+    "dev": "lerna run --stream prepack -- --incremental && lerna --ignore=@logto/integration-test run --parallel dev",
```

Comment:
Is this `-- --` expected?

Reply:
yes, the first `--` indicates passing the following args to the command